src/controllers/user.controller.ts

```diff
@@ -0,0 +1,34 @@
+import { Response } from 'express';
+import * as UserService from '../services/user.service';
+import RequestType from '../types/requestType';
+
+export const info = async (req: RequestType, res: Response) => {
+  const { userId } = req;
+  const { username } = req.body;
+  if (!userId) {
+    return res.status(401).json({
+      success: false,
+      message: 'Unauthorized',
```

Comment:
cái này cũng vẫn phải là lỗi User Not Found chứ nhỉ ?
Unauthorized là không log in.